src/test/java/core/basesyntax/service/FileServiceImplTest.java

```diff
@@ -0,0 +1,149 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileServiceImplTest {
+    private static FileService fileService;
+
+    @BeforeClass
+    public static void beforeClass() {
+        fileService = new FileServiceImpl();
+    }
+
+    @Test //(expected = RuntimeException.class)
+    public void readFromFileNullInputPath_NotOk() {
+        String path = null;
+
+        try {
+            fileService.readFromFile(path);
+        } catch (RuntimeException e) {
+            return;
+        }
+        fail("RuntimeException should be thrown if value is null");
+    }
+
+    @Test
+    public void readFromFileInputEmptyLinePath_NotOk() {
+        String path = "";
+
+        try {
+            fileService.readFromFile(path);
+        } catch (RuntimeException e) {
+            return;
+        }
+        fail("RuntimeException should be thrown if value is null");
+    }
+
+    @Test
+    public void readFromFileInputIncorrectLinePath_NotOk() {
+        String path = "-";
```

Comment:
both test checks the behavior of read from an incorrect path. leave only one test

Reply:
> both test checks the behavior of read from an incorrect path. leave only one test

fixed)